examples/mockups/index.html

```diff
@@ -0,0 +1,329 @@
+          
+              <h2 class="ui header">How it works</h2>
+
+              <div class="ui grid stackable three column">
+
+                <div class="column center aligned">
+                  <div class="ui image">
+                    <img src="images/infographic-01.png">
+                  </div>
+                  <p>
+                    Write your documentation easily, using .md or .rst, directly in your own IDE, as you work on what maters: your project.
+                  </p>
+                </div>
+                <div class="column center aligned">
+                  <div class="ui image">
+                    <img src="images/infographic-02.png">
+                  </div>
+                  <p>
+                    Upload your work to your favorite source code repository hosting service. We support GitHub, GitLab, BitBucket and more.
+                  </p>
+                </div>
+                <div class="column center aligned">
+                  <div class="ui image">
+                    <img src="images/infographic-03.png">
+                  </div>
+                  <p>
+                    Setup your account with Read the Docs, add a .config file to tell us your preferences and we'll do the rest!
+                  </p>
+                </div>
+
+              </div>
+
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+    </section>
+
+
+    <!-- It works -->
+    <section class="ui vertical segment very padded" id="it-works">
+
+      <div class="ui container">
+        <div class="ui grid center aligned">
+
+          <div class="row">
+            <div class="column twelve wide computer sixteen wide tablet">
+          
+              <p class="shout text">
+                We generate, host and manage your documentation versions for you, outputting both html and pdf formats. 
+                Whenever you push code, we'll build your docs.
+              </p>
+              
+              <p class="shout text">
+                <big>It Just works.</big>
+              </p>
+
+            </div>
+          </div>
+
+        </div>
+      </div>
+
+    </section>
+
+    <!-- end Contents
+    ----------------------------------- -->
+
+
+    <!-- Footer 
+    ----------------------------------- -->
+    <footer id="footer">
+        
+      <section class="main">
+        <div class="ui container">
+
+          <div class="connections">
+
+            <div class="links">
+
+              <ul>
+                <li class="head">Product</li>
+                <li><a href="product.html">How it works</a></li>
+                <li><a href="pricing.html">Pricing</a></li>
+                <li><a href="https://readthedocs.com/">for business</a></li>
+                <li><a href="https://www.ethicalads.io/advertisers/?ref=rtd">Advertise with Us</a></li>
+              </ul>
+              
+              <ul>
+                <li class="head">Company</li>
```

Comment:
The reason I used the heading inside the a `li` is so that I could style it with the exact same height as any other `li` element in the `.footer .links`. That way they are interchangeable without losing the horizontal alignment to sibling `ul`s that might have more headings or headings in different positions.

_For example:_
![link-alignment](https://user-images.githubusercontent.com/4049894/142889000-aedea441-0399-42d8-9091-e8724a7f40d2.jpg)

Reply:
Another thing to consider here is that we talked about having a much smaller link list here. If that is the case then it probably doesn't make sense to account for interchangeable headings and we can compromise more easily to just one heading per column.

Reply:
> The reason I used the heading inside the a li is so that I could style it with the exact same height as any other li element

That will get thrown off by any multiline items in the lists too though. I'd say this is minor either way, but agree we should be able to accomplish single heading columns.

Instead of reworking things, is there any reason to not just fork the work I did on this on the application footer?

I'd say it's not a big concern either way. We should be able to

Reply:
> That will get thrown off by any multiline items in the lists too though.
Line height and margins should work together to not let that happen by any random number. i.e. 0.5 is acceptable as it catches up on the next off line.

> Is there any reason to not just fork the work I did on this on the application footer?
No reason that I can see. We just didn't talk about this before. I'll do that.

Reply:
👍 Imported footer from app and inverted its outer `.segment` to match the color we were going for here.